src/client.js

```diff
@@ -139,13 +139,27 @@ Client.prototype.connect = function(args) {
 	var webirc = null;
 	var channels = [];
 
-	if (args.join) {
-		var join = args.join.replace(/\,/g, " ").split(/\s+/g);
-		join.forEach(function(chan) {
-			channels.push(new Chan({
-				name: chan
-			}));
+	if (args.channels) {
+		channels = args.channels.map(function(chan) {
+			if (!chan.name) {
+				return;
```

Comment:
We probably should print an error message there to let the user know that a channel has been ignored instead of just aborting like that.